src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurSAEFileFixture.java

```diff
@@ -37,7 +37,8 @@
     DOCUMENT_NUMBER_TEST(4, 200.00),
     EMPLOYEE_NAME_TEST(4, 200.00),
     CASH_AND_CARD_TEST(2, 105.00),
-    CANCELED_TRIP_TEST(2, -432.40);
+    CANCELED_TRIP_TEST(2, -432.40),
+    PDP_TEST(3, 1000);

```

Comment:
Since you have four SAE detail lines for your PDP test, the first arg should be a 4 instead of a 3. (I might refactor this enum in the future so that the line count is derived instead of explicitly given here.)